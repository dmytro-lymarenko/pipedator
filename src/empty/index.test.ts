import { empty } from './index';

describe('empty()', () => {
	it('should validate an empty string', () => {
		expect(empty().validate('')).toBeNull();
	});

	it('should validate an empty array', () => {
		expect(empty().validate([])).toBeNull();
	});

	it('should support custom message', () => {
		expect(empty('Custom message').validate('not empty string')).toMatchSnapshot();
	});
});
