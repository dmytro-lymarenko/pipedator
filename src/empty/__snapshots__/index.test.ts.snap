// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty() should support custom message 1`] = `
Object {
  "errors": Array [
    Object {
      "message": "Custom message",
      "path": Array [],
      "value": "not empty string",
    },
  ],
  "rootValue": "not empty string",
}
`;
